### PR TITLE
AC-1509: Remove outdated JSCS static tests from magento

### PR DESCRIPTION
**Pull request by loginesta**
https://jira.corp.magento.com/browse/AC-1509 Remove outdated JSCS static tests from magento

### CODE REVIEW

File: eslint/rules/jquery-no-trim.js [hunk 1]
```diff
@@ -29,13 +29,13 @@ module.exports = {
              * @param {Object} node - The node to check.
              */
             CallExpression: function (node) {
-                // jscs:disable requireCurlyBraces
+                /* eslint-disable curly */
```
Comment (sivaschenko) [inline eslint/rules/jquery-no-trim.js:32]: Would you be able to fix the code style to match the standard and remove the disable/enable annotation completely?

Comment (loginesta) [reply eslint/rules/jquery-no-trim.js:32]: sure! no big deal

**Review by sivaschenko — APPROVED** [review]

**Comment by sivaschenko** [comment]
New rules have been tested on magento2 codebase and resulted in 0 errors (preserved compatibility with previous JSCS configuration)

**Comment by sivaschenko** [comment]
@magento import pr to magento-commerce/magento-coding-standard

**Comment by m2-github-services** [comment]
@sivaschenko an error occurred during the Pull Request import.

**Pull request merged**